docs/reference/filebeat/filebeat-input-filestream.md

```diff
@@ -325,57 +325,82 @@ The setting relies on the modification time of the file to determine if a file i
 
 To remove the state of previously harvested files from the registry file, use the `clean_inactive` configuration option.
 
-## Take over [filebeat-input-filestream-take-over]
+#### `take_over` [filebeat-input-filestream-take-over]
+
+```{applies_to}
+stack: beta
```

Comment:
Ugh. The formatting of an `applies_to` inside a `suggestion`  got me...

````suggestion
```{applies_to}
stack: beta
```
````